conf/messages.en

```diff
@@ -1481,13 +1482,51 @@ settlorBusinessName.error.length = Business’s name must be 105 characters or l
 settlorBusinessName.error.invalid = Business name must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
 
 settlorBusinessUtrYesNo.checkYourAnswersLabel = Do you know {0}’s Unique Taxpayer Reference (UTR) number?
+settlorBusinessUtrYesNo.title = Do you know the business’s Unique Taxpayer Reference (UTR) number?
+settlorBusinessUtrYesNo.heading = Do you know {0}’s Unique Taxpayer Reference (UTR) number?
+settlorBusinessUtrYesNo.error.required = Select yes if you know the business’s Unique Taxpayer Reference (UTR) number
+
 settlorBusinessUtr.checkYourAnswersLabel = What is {0}’s Unique Taxpayer Reference (UTR) number?
+settlorBusinessUtr.title = What is the business’s Unique Taxpayer Reference (UTR) number?
+settlorBusinessUtr.heading = What is {0}’s Unique Taxpayer Reference (UTR) number?
```

Comment:
Prototype doesn't include **number**. I can't recall which one we decided to go with

Reply:
removed now

Reply:
We go with number :)

Reply:
added back in now